exercises/grep/canonical-data.json

```diff
@@ -204,6 +204,37 @@
             "files": ["iliad.txt"]
           },
           "expected": []
+        },
+        {
+          "description": "One file, one match, line number and file flags",
+          "property": "grep",
+          "input": {
+            "pattern": "ten",
+            "flags": ["-n", "-l"],
+            "files": ["iliad.txt"]
+          },
+          "expected": [
+            "2:His wrath pernicious, who ten thousand woes"
+          ]
+        },
+        {
+          "description": "One file, several matches, inverted and match entire lines flags",
+          "property": "grep",
+          "input": {
+            "pattern": "Illustrious into Ades premature",
```

Comment:
this should have a comma inside the string. Otherwise, since the flag to match entire lines and the line in the file is `"Illustrious into Ades premature,"` the line will be included in the output.

Reply:
Exactly, well spotted!